python/lsst/meas/extensions/trailedSources/NaivePlugin.py

```diff
@@ -227,21 +228,21 @@ def measure(self, measRecord, exposure):
         # Error in theta
         dThetadIxx = -Ixy / (xmy2 + 4.0*xy2)  # dThetadIxx = -dThetadIyy
         dThetadIxy = xmy / (xmy2 + 4.0*xy2)
-        thetaErr = np.sqrt(dThetadIxx*dThetadIxx*(IxxErr2 + IyyErr2) + dThetadIxy*dThetadIxy*IxyErr2)
+        thetaErr = sqrt(dThetadIxx*dThetadIxx*(IxxErr2 + IyyErr2) + dThetadIxy*dThetadIxy*IxyErr2)
 
         # Error in flux
         dFdxc, dFdyc, _, dFdL, dFdTheta = gradFlux
-        fluxErr = np.sqrt(dFdL*dFdL*lengthErr*lengthErr + dFdTheta*dFdTheta*thetaErr*thetaErr
-                          + dFdxc*dFdxc*xcErr2 + dFdyc*dFdyc*ycErr2)
+        fluxErr = sqrt(dFdL*dFdL*lengthErr*lengthErr + dFdTheta*dFdTheta*thetaErr*thetaErr
+                        + dFdxc*dFdxc*xcErr2 + dFdyc*dFdyc*ycErr2)
```

Comment:
There appears to be an extraneous space here.

Reply:
Deleted extra space